README.md

```diff
@@ -1,95 +1,270 @@
+```
 
-#Create database (Only the first time)
-createdb <DatabaseName>
+This will create a migration file inside the `migrations/` directory containing the migration setup and rollback procedures.
 
-# Update env. The DEFAULT_USER_ID is important
+#### Manually applying migrations
 
-# Update state
-npm i
-npm run migrate up
+To run migrations manually:
 
-# Only once
-npx ts-node ./scripts/parseS3Pools.ts
-npx ts-node ./scripts/updateProjectThumbnails.ts
+```bash
+npm run migrate up
+```
 
-# On each asset pack change
-npm run seed
+To rollback migrations manually:
 
-# Start
-npm start
+```bash
+npm run migrate down
 ```
 
-# Rewrites to S3
+## Getting Started
 
-- `GET /v1/storage/assetPacks/:filename` => https://s3.amazonaws.com/nico.decentraland.zone/asset_packs/:filename (for the thumbnail, :id.png)
-- `GET /v1/storage/contents/:hash` => https://s3.amazonaws.com/nico.decentraland.zone/contents/:hash
+### Prerequisites
 
-# Also S3 but behind auth
+Before running this service, ensure you have the following installed:
 
-- `GET /v1/projects/:id/manifest` => https://s3.amazonaws.com/nico.decentraland.zone/projects/:id/manifest.json
-- `GET /v1/pools/:id/manifest` => https://s3.amazonaws.com/nico.decentraland.zone/projects/:id/pool.json
-- `GET /v1/projects/:id/media/:filename` => https://s3.amazonaws.com/nico.decentraland.zone/projects/:id/filename
+- **Node.js**: Check `.nvmrc` for the required version
+- **npm**: Comes with Node.js
+- **PostgreSQL**: Version 12.x or higher
+- **Docker**: For containerized external services (optional)
 
-Take into account that `/v1/` correspond to the version that you specify in the file `.env` with `API_VERSION`
+### Installation
 
-# S3 structure
+1. Clone the repository:
 
 ```bash
-projects
-|____PROJECT_ID
-| |____manifest.json
-| |____pool.json
-| |____east.png
-| |____north.png
-| |____preview.png
-| |____south.png
-| |____thumbnail.png
-| |____west.png
-asset_packs
-|____ASSET_PACK_ID.png
-contents
-|____HASH1
-|____HASH2
-|____HASH3
+git clone https://github.com/decentraland/builder-server.git
+cd builder-server
 ```
 
-#Extra Info
-If you are using windows subsystem, you will need to start the postgresql service each time
+2. Install dependencies:
 
-`sudo service postgresql start`
+```bash
+npm install
+```
 
-# Running external services locally with Docker Compose
+3. Build the project:
 
-If you have docker running on your machine and want to have external dependencies running locally on your machine,
-you can use the `docker-compose` file present in the repo to do it in a simple and centralized way and avoid the hassle of
-configuring each one of them independantly.
+```bash
+npm run build
+```
+
+### Configuration
+
+The service uses environment variables for configuration. Create a `.env` file in the root directory based on `.env.example`:
+
+```bash
+cp .env.example .env
+```
 
-First, run all services simultaneously with:
+Key environment variables:
```

Comment:
Would you mind creating a `.default.env` with all the required variables to run the service and removing this section so there's a single place where all of them live? Would you also mind to adding comments to these variabrles as they're described here in the description column?